scipy/stats/stats.py

```diff
@@ -5945,16 +5945,22 @@ def ttest_ind(a, b, axis=0, equal_var=True, nan_policy='propagate',
     against the null hypothesis of equal population means.
 
     By default, the p-value is determined by comparing the t-statistic of the
-    observed data against a theoretical t-distribution, which assumes that the
-    populations are normally distributed.
```

Comment:
@jb-leger I got this wrong originally; t-tests are more flexible than that. References often say something like "The means of the two populations being compared should follow normal distributions." I do not have much formal statistics education, but my understanding is that the mean of a population usually has a definite value rather than a distribution, so this sort of statement, strictly interpreted, does not make sense. Is my attempt at a more precise rephrasing correct?

>   which assumes that means _of samples from the underlying populations_ are normally distributed.

Reply:
For considering the t-test, we have two hypothesis:
 - empirical means follow normal distribution
 - empirical variance follows χ² distribution

The two hypothesis are important. A sufficient condition is the normality of samples. Your formulation forget the second hypothesis.

However, in real life, if the second hypothesis is not verified, this does not have huge influence on the results, and the t-test is often applied without checking the second hypothesis. In real life, only the first hypothesis (normality of empirical means, a.k.a. normality means of samples) is checked. In fact, when your data is not-normal, the first hypothesis fails first, and it is very difficult to found data that check the first hypothesis without checked the second.

Reply:
Right, I know about both conditions. I didn't mean to list the conditions for a t-test, I only meant to list the ones that might contribute to the user wanting to use a permutation test. Maybe I will just remove the statement.

Reply:
@mdhaber, I know. I ask myself why you ask the conditions… For users, I think the condition about the means of samples is enough.

But, for permutation test, a condition you do not explain is exchangeability. For me the most useful case of permutation test, is when independence is not checked, but when we have exchangeability. Maybe a formulation like « permutation test does not require normality of empirical means and independance of samples, but only exchangeability of samples. »

Reply:
I think for now I'll just remove the erroneous comment.